pkg/cmd/roachtest/tests/ruby_pg_blocklist.go

```diff
@@ -204,6 +202,8 @@ var rubyPGIgnorelist = blocklist{
 	`PG::Connection handles server close while asynchronous connect`:                                                                                                               "flaky",
 	`PG::Connection OS thread support Connection.new shouldn't block a second thread`:                                                                                              "flaky",
 	`PG::Connection multinationalization support respect and convert character encoding of input strings should convert error string to #put_copy_end`:                             "unknown",
+	`PG::Connection in nonblocking mode rejects to send lots of COPY data`:                                                                                                         "flaky",
+	`running with sync_* methods PG::Connection in nonblocking mode rejects to send lots of COPY data`:                                                                             "flaky",
```

Comment:
i feel like we may as well add that 3rd test here too, since it's just as likely to flake on this branch